pkg/util/ecs/common_test.go

```diff
@@ -0,0 +1,301 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+// +build docker
+
+package ecs
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/util/containers"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// dummyECS allows tests to mock a ECS's responses
+type dummyECS struct {
```

Comment:
Given this mock is also used by ecs_test.go, it may be better to structure this a bit and put it somewhere else, for instance into a testutil subpackage or something of the sort?

Reply:
totally agree 👍